### PR TITLE
Add organizations

### PR DESCRIPTION
**Pull request by MarioRodrigues10**

### CODE REVIEW

File: priv/repo/seeds/departments.exs [hunk 1]
```diff
@@ -2,18 +2,22 @@ defmodule Atomic.Repo.Seeds.Departments do
...
 
+
   def seed_departments() do
     case Repo.all(Department) do
       [] ->
         Department.changeset(
           %Department{},
           %{
             name: "Merchandise and Partnerships",
+            # get organization id
```
Comment (RuiL1904) [inline priv/repo/seeds/departments.exs:19]:
```suggestion
```

File: priv/repo/seeds/organizations.exs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        |> Repo.insert!()
+
+        Organization.changeset(
+          %Organization{},
+          %{
+            name: "Atomic",
+            description: "Atomic is a company that provides a platform for the development of digital products and services.",
+            location: %{
+              name: "Atomic",
+              url: "https:// atomic.pt",
```
Comment (RuiL1904) [inline priv/repo/seeds/organizations.exs:33]:
```suggestion
              url: "https://atomic.pt",
```

**Review by RuiL1904 — COMMENTED** [review]
It's currently not possible to add or delete a department from an organization. Take care of that 🙏🏼

File: priv/repo/migrations/2022000000000_create_organizations.exs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+defmodule Atomic.Repo.Migrations.CreateOrganizations do
+  use Ecto.Migration
+
+  def change do
+    create table(:organizations, primary_key: false) do
+      add :id, :binary_id, primary_key: true
+      add :name, :string, null: false
+      add :description, :string, null: false
+      add :location, :map
```
Comment (feliciofilipe) [inline priv/repo/migrations/2022000000000_create_organizations.exs:9]: why do we need a location?

Comment (MarioRodrigues10) [reply priv/repo/migrations/2022000000000_create_organizations.exs:9]: To see where an organization is located (Gualtar, Azurém, ...)

**Review by feliciofilipe — APPROVED** [review]

**Pull request merged**